src/sseq.rs

```diff
@@ -458,6 +543,45 @@ mod sseq_test {
         );
     }
 
+    #[test]
+    fn test_remove() {
+        let mut seq = SSeq::from_bytes(b"GCAT");
+        assert_eq!(seq.remove(2), b'A');
+        assert_eq!(seq, SSeq::from_bytes(b"GCT"));
+    }
+
+    #[test]
+    fn test_insert() {
```

Comment:
It might be nice to add test cases that assert that we panic if we provide invalid data, such as insufficient capacity.  Basically to enforce under future refactoring that we catch these cases instead of silently doing the wrong thing.

Reply:
Done!